src/PlPage/TrackList.jsx

```diff
@@ -0,0 +1,25 @@
+import TrackItem from './TrackItem';
+
+const TrackList = ({ musicSet = [], isEdit, isDelete, trackCount }) => {
+  if (!musicSet || musicSet.length === 0) {
```

Comment:
props에서 musicSet=[] 이라는 기본 값을 주었기 때문에 !musicSet을 삭제해도 좋을 것 같습니다. 다른 방법으로는 musicSet에 기본값을 주지 않는 방법이 있겠네요
```suggestion
  if (musicSet.length === 0) {
```

Reply:
466d00e866e8ddbe6c1af8ebba263becf16bd431